validator/json_schemas/scanning.schema.json

```
@@ -0,0 +1,17 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "scanning.schema.json",
+  "title": "Scanning Content",
+  "type": "object",
+  "properties": {
```

to prevent typos that validate:
```suggestion
  "additionalProperties": false,
  "properties": {
```